src/d_main.c

```diff
@@ -2637,7 +2637,15 @@ void D_DoomMain(void)
 
   MN_InitMenuStrings();
 
-  if (startloadgame >= 0)
+  // Auto save slot is 255 for -loadgame command.
+  if (startloadgame == 255 && !demorecording && gameaction != ga_playdemo
+      && !netgame)
```

Comment:
`255` was chosen in case the auto save feature is ever expanded in the future to support multiplayer, where net packets send the save game slot as a byte: https://github.com/fabiangreffrath/woof/blob/master/src/net_structrw.c#L78

Reply:
Very foresightful. 👍